core/src/test/java/org/elasticsearch/index/query/FuzzyQueryBuilderTests.java

```diff
@@ -107,7 +113,12 @@ public void testUnsupportedFuzzinessForStringType() throws IOException {
     }
 
     public void testToQueryWithStringField() throws IOException {
-        assumeTrue("test runs only when at least a type is registered", getCurrentTypes().length > 0);
+        try {
+            assumeTrue("test runs only when at least a type is registered", getCurrentTypes().length > 0);
```

Comment:
Unfortunately @ConditionalIgnore doesn't seem to be part of junits standard set of annotations yet (seems to be targeted for junit 5). Rather than build our own or import one of the custom solutions I found, I'd keep the current solution. This whole test class is going away with the deprecated query builder anyway soon (will stay on the 5.x branch for it's lifetime however).